packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -46,20 +50,17 @@ const useStyles = makeStyles({
 });
 
 // this is a stub to move the logic here that should end up here.
-export const ChatList = (
-  props: MgtTemplateProps &
-    IChatListItemInteractionProps &
-    IChatListMenuItemsProps & {
-      buttonItems?: ChatListButtonItem[];
-      chatThreadsPerPage: number;
-    }
-) => {
+export const ChatList = ({
+  lastReadTimeInterval = 30000, // default to 30 seconds
```

Comment:
is this intended to be configurable? if not, would it make sense to define a const in the class method?

Reply:
It is configurable, this just defaults it.